components/workflow.rst

```diff
@@ -44,19 +44,23 @@ these statuses are called **places**. You can define the workflow like this::
     use Symfony\Component\Workflow\Workflow;
     use Symfony\Component\Workflow\MarkingStore\SingleStateMarkingStore;
 
-    $builder = new DefinitionBuilder();
-    $builder->addPlaces(['draft', 'review', 'rejected', 'published']);
-
-    // Transitions are defined with a unique name, an origin place and a destination place
-    $builder->addTransition(new Transition('to_review', 'draft', 'review'));
-    $builder->addTransition(new Transition('publish', 'review', 'published'));
-    $builder->addTransition(new Transition('reject', 'review', 'rejected'));
-
-    $definition = $builder->build();
+    $definition = new DefinitionBuilder()
+        ->addPlaces(['draft', 'review', 'rejected', 'published'])
+        // Transitions are defined with a unique name, an origin place and a destination place
+        ->addTransition(new Transition('to_review', 'draft', 'review'))
+        ->addTransition(new Transition('publish', 'review', 'published'))
+        ->addTransition(new Transition('reject', 'review', 'rejected'))
+        ->build()
+    ;
 
     $marking = new SingleStateMarkingStore('currentState');
     $workflow = new Workflow($definition, $marking);
 
+.. versionadded:: 3.3
+    The fluent interface for ``DefinitionBuilder`` class was added in Smyfony 3.3.
+    Before you had to call ``addPlaces()``, ``addTransition()`` and ``build()``
```

Comment:
[...] had to call the [...]

Reply:
but,the example occur "Fatal error: Class 'Symfony\Component\Workflow\DefinitionBuilder' not found ",i use the symfony 3.3.10 version,why?

Reply:
The class is there: https://github.com/symfony/symfony/blob/v3.3.10/src/Symfony/Component/Workflow/DefinitionBuilder.php

Are you sure that you use Symfony 3.3.10? I suggest that you use one of [the support channels](https://symfony.com/support) to seek help with your problem.